website/pages/blog/announcing-cloudquery-new-hub.mdx

```diff
@@ -23,7 +23,7 @@ We are excited to announce that we have launched [CloudQuery Hub](https://hub.cl
 
 To protect the CloudQuery Infrastructure that hosts plugins, we are going to introduce rate limiting per IP address when downloading plugins without logging in. If you frequently download plugins at a high rate, we recommend to either cache the downloaded assets or to log in to increase your rate limit.
 
-This change is not coming immediately. First, we are going to enable account creation and add support for authenticating and building production deployments. After that, for an initial one-month transition period, the rate-limiting will be disabled and plugins will be published to both CloudQuery Hub and GitHub. Thereafter the default rate limits will apply and new plugin versions will only be published to CloudQuery Hub.
+Previous versions of plugins are published on GitHub and will be hosted there, so you should not experience any disruption to your production systems. New plugin versions will be continued to published to both GitHub and CQ Hub so you will have time to create an account for higher rate limits. 
```

Comment:
Should we specify for how long we'll keep publishing to GitHub?

Reply:
Let's not commit yet.